packages/terra-form-input/src/terra-dev-site/doc/example/controlled/InputProgrammaticSet.jsx

```diff
@@ -0,0 +1,51 @@
+import React, { useState, useRef } from 'react';
+import classNames from 'classnames/bind';
+import Button from 'terra-button';
```

Comment:
terra-button is not a dependency of this package, a native button should be used here to remove the dependency import

Reply:
Removed it here https://github.com/cerner/terra-core/pull/3135/commits/17a5bc400687a7f9b1b945eb45b51f42adbb27f6